terraform/helm/aptos-node/templates/validator.yaml

```diff
@@ -87,6 +87,8 @@ spec:
           value: {{ .remoteLogAddress }}
         {{- end }}
       {{- end }}
+        - name: KUBERNETES_NAMESPACE
```

Comment:
there is no need to pass the namespace from helm.
We should use the k8s downward API for that:

```
 - name: KUBERNETES_NAMESPACE
          valueFrom:
            fieldRef:
              fieldPath: metadata.namespace
```

https://kubernetes.io/docs/tasks/inject-data-application/environment-variable-expose-pod-information/#the-downward-api

Reply:
Is there a difference? This file is meant to be helm templated anyways

Reply:
it's just more common practice and i think it's better to not template this if we don't have to.